djweb/simulations/static/js/assembly_addition.js

```diff
@@ -1,6 +1,12 @@
 const computer_size = 7;
 const num_code_lines = 4;
 const highlighted_registers = [6];
+const line_descriptions = [
+    "<b>CLA 4: </b> Clears the accumulator and stores the value at address 4 (which you'll see if you mouseover is 12) into the accumulator.<br />",
+    "<b>ADD 5: </b> Adds the value at address 5 (30) to the accumulator.<br />",
+    "<b>STO 6: </b> Stores the value of the accumulator into the register at address 6.<br />",
+    "<b>HTR: </b> Tells the computer to stop.<br />",
+];
```

Comment:
From below (I don't know how to properly access it):
```
    computer.general_memory[4].fixed_point = 12;
    computer.general_memory[5].fixed_point = 30;
```
Why aren't these lines part of the program? Right now, when I press clear and assemble, memory slots 4 and 5 become 0.

Reply:
This demo was written before you could define numerical constants in the assembly code; now I'm wondering if we should just have this demo hardcode the values into the computer before introducing DEC and ORG in the floating point demo.  @alanabreu106 what do you think?